integration-tests/src/test/java/oracle/kubernetes/operator/ITPodsRestart.java

```diff
@@ -305,6 +322,207 @@ public void testServerPodsRestartByChangingResource() throws Exception {
+      logger.info(exec.stdout());
+
+      logger.info("Verifying if the admin server is terminating");
+      verifyPodStatus(podName, "Terminating");
+      verifyPodStatus(podName, "Running");
+
+    } finally {
+      logger.log(
+          Level.INFO, "Reverting back the domain to old crd\n kubectl apply -f {0}", originalYaml);
+      TestUtils.exec("kubectl apply -f " + originalYaml);
+      logger.info("Verifying if the admin server is terminating");
+      verifyPodStatus(podName, "Terminating");
+      verifyPodStatus(podName, "Running");
+    }
+    logger.log(Level.INFO, "SUCCESS - {0}", testMethodName);
+  }
+
+  /**
+   * Add restartVersion:v1.1 at cluster level and verify the managed servers pods are Terminated and
+   * recreated
+   *
+   * @throws Exception when domain.yaml cannot be read or modified to include the
+   *     restartVersion:v1.1
+   */
+  @Test
+  public void testClusterRestartVersion() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    String podName = domainUid + "-managed-server1";
+
+    try {
+      // Modify the original domain yaml to include restartVersion in admin server node
+      DomainCRD crd = new DomainCRD(originalYaml);
+      Map<String, String> cluster = new HashMap();
+      cluster.put("restartVersion", "v1.1");
+      crd.addObjectNodeToCluster("cluster-1", cluster);
+      String modYaml = crd.getYamlTree();
+      logger.info(modYaml);
+
+      // Write the modified yaml to a new file
+      Path path = Paths.get(restartTmpDir, "restart.cluster.yaml");
+      logger.log(Level.INFO, "Path of the modified domain.yaml :{0}", path.toString());
+      Charset charset = StandardCharsets.UTF_8;
+      Files.write(path, modYaml.getBytes(charset));
+
+      // Apply the new yaml to update the domain crd
+      logger.log(Level.INFO, "kubectl apply -f {0}", path.toString());
+      ExecResult exec = TestUtils.exec("kubectl apply -f " + path.toString());
+      logger.info(exec.stdout());
+      logger.info("Verifying if the cluster is restarted");
+      verifyPodStatus(podName, "Terminating");
+      verifyPodStatus(podName, "Running");
+    } finally {
+      logger.log(
+          Level.INFO, "Reverting back the domain to old crd\n kubectl apply -f {0}", originalYaml);
+      TestUtils.exec("kubectl apply -f " + originalYaml);
+      logger.info("Verifying if the cluster is restarted");
+      verifyPodStatus(podName, "Terminating");
+      verifyPodStatus(podName, "Running");
+    }
+    logger.log(Level.INFO, "SUCCESS - {0}", testMethodName);
+  }
+
+  /**
+   * Add restartVersion:v1.1 at managed server level and verify the managed server pod are
+   * Terminated and recreated
+   *
+   * @throws Exception when domain.yaml cannot be read or modified to include the
+   *     restartVersion:v1.1
+   */
+  @Test
```

Comment:
comment out the failing test with bug number and update the bug with instructions to run the test....just comment out @Test line

Reply:
Done.